plugin/fzf.vim

```diff
@@ -506,7 +518,7 @@ function! s:execute(dict, command, use_height, temps) abort
   endif
   if s:is_win
     let batchfile = s:fzf_tempname().'.bat'
```

Comment:
Unrelated to the issue, shouldn't we have to add this to `temps` so that it's deleted at the end? (`let temps.batchfile = ...`)